README.md

```diff
@@ -16,6 +16,24 @@ Elemental is a tool for installing, configuring and updating operating system im
 *   **Updates:** Update an existing OS installation from a newer image.
 *   **Extensibility:** Extend the OS installation image with extensions.
 
+## Quick-start
+
+Elemental can be used to convert an OCI image into a bootable disk-image. This requires that the OCI image actually contains a bootloader, kernel, initrd and init-system. In this quick-start we will use the tumbleweed example.
+
+```sh
+$ make # build elemental binaries
+$ qemu-img create -f raw build/elemental-tumbleweed.img 10G
+$ sudo losetup --show -f build/elemental-tumbleweed.img # make a note of the device-name
+$ sudo ./build/elemental3-toolkit install --os-image=registry.opensuse.org/devel/unifiedcore/tumbleweed/containers/uc-base-os-kernel-default:0.0.1 --config=examples/tumbleweed/config.sh --target /dev/loopX # use the loopback-device printed in previous step.
```

Comment:
@dirkmueller Can we get a `:latest` build of this rather than a specific number? I feel like we're going to frequently forget to update this.

Reply:
@jdob sure, there is a ':latest` tag alongside the tumbleweed versioning as well.